apps/design/backend/migrations/1756159958924_organizations-table.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { ManagementClient } = require('auth0');
+const { assertDefined } = require('@votingworks/basics');
+
+/**
+ * @type {import('node-pg-migrate').ColumnDefinitions | undefined}
+ */
+exports.shorthands = undefined;
+
+/**
+ * @param pgm {import('node-pg-migrate').MigrationBuilder}
+ * @param run {() => void | undefined}
+ * @returns {Promise<void>}
+ */
+exports.up = async (pgm) => {
+  pgm.createTable('organizations', {
+    id: { type: 'text', primaryKey: true },
+    name: { type: 'text', notNull: true, unique: true },
+  });
+
+  if (process.env.NODE_ENV === 'production' || process.env.AUTH_ENABLED) {
```

Comment:
The `pnpm db:reset-dev` command runs all of the migrations, so we want to make sure that still works even if we're not hooked up to Auth0 in dev